openapi.yaml

```diff
@@ -6133,6 +6133,9 @@ paths:
                       example: true
                     interfaces:
                       $ref: '#/components/schemas/LinodeConfigInterfaces'
+                    firewall_id:
```

Comment:
Agreed on this suggestion -- I've put this property in a separate PR in case of earlier activation.
https://github.com/linode/linode-api-docs/pull/871